test/e2e/framework/syncer.go

```diff
@@ -511,6 +515,256 @@ func (sf *syncerFixture) Start(t *testing.T) *StartedSyncerFixture {
 	return startedSyncer
 }
 
+// BootstrapSyncTarget simulate the start of a Syncer and performs what the Syncer does to bring and maitain
+// the SyncTarget to a Ready status (heartbeat, and API schema compatibility).
+// No resource will be without effectively synced after calling this method.
+func (sf *syncerFixture) BootstrapSyncTarget(t *testing.T) *StartedSyncerFixture {
```

Comment:
strange name if this simulates only

Reply:
Renamed it to `SimulateStartWithoutSyncing`